queue_services/entity-emailer/src/entity_emailer/worker.py

```diff
@@ -74,6 +74,20 @@ async def publish_event(payload: dict):
 
 def send_email(email: dict, token: str):
     """Send the email."""
+    # stop processing email when payload is incompleted.
+    if not email \
+            or 'recipients' not in email \
+            or 'content' not in email \
+            or 'body' not in email['content']:
+        logger.debug('Send email: email object(s) is empty')
+        raise QueueException('Unsuccessful sending email.')
```

Comment:
Can we update the message we are using to instantiate `QueueException`?  To maybe something like "Unsuccessful sending email - required email object(s) is missing"?  This way the `message_processing.last_error` field will provide a bit more detail as to what the error was.  And same thing for QueueException call below on line 89.